src/app/portlets/content-types/form/content-types-form.component.html

```diff
@@ -1,77 +1,108 @@
         <div class="ui-g">
             <div class="ui-g-6">
                 <div class="form__group content-type__form-title">
-                    <i class="fa" [ngClass]="templateInfo.icon"></i>
-                    <input pInputText type="text" id="content-type-form-name" (focus)="formState === 'collapsed' && toggleForm()" [placeholder]="templateInfo.placeholder" name="name" formControlName="name" tabindex="1" #name class="mousetrap" />
-                    <field-validation-message [message]="i18nMessages['contenttypes.form.name.error.required']" [field]="form.get('name')"></field-validation-message>
+                    <strong>{{form.value.name}}</strong>
                 </div>
             </div>
             <div class="ui-g-6" *ngIf="isEditMode()">
                 <div class="content-type__form-actions">
-                    <button type="button" pButton flat label="Edit" (click)="editForm()" icon="fa-edit" id="form-edit-button" [disabled]="formState === 'expanded'"></button>
+                    <button type="button" pButton flat label="Edit" (click)="editForm()" icon="fa-edit" id="form-edit-button"></button>
                 </div>
             </div>
         </div>
+        <div>
+            <span class="content-type__form-variable" *ngIf="isEditMode()">
+                {{i18nMessages['contenttypes.content.variable']}}:
+                {{data.variable }} •
+            </span>
+            <span class="content-type__form-identifier" *ngIf="isEditMode()">
+                {{i18nMessages['contenttypes.form.identifier']}}:
+                {{data.id }}
+            </span>
+        </div>
     </div>
-    <div class="content-type__full-form" [class.content-type__full-form--expanded]="formState === 'expanded'" [@enterAnimation]="formState">
-        <div class="ui-g">
-            <div class="ui-g-6">
-                <div class="form__group" *ngIf="isEditMode()">
-                    <b>{{i18nMessages['contenttypes.content.variable']}}:</b>
-                    <label>{{data.variable }}</label>
-                </div>
-                <div class="form__group">
-                    <label for="content-type-form-description" class="form__label">{{i18nMessages['contenttypes.form.label.description']}}</label>
+    
+    <p-dialog
+        modal="modal"
+        width="500"
+        [(visible)]="show"
+        [responsive]="true"
+        [resizable]="false"
+        [draggable]="false"
+        [closable]="false"> 
+    
+        <p-header class="container-fluid">
+            {{templateInfo.header}} {{i18nMessages['contenttypes.content.contenttype']}}
+        </p-header>
+    
+        <form [formGroup]="form" class="content-type__form container-fluid" (ngSubmit)="submitContent($event)" id="content-type-form" #formEl="ngForm" novalidate>
+            <div class="form__group">
+                <!-- TODO: Fix md-inputtext directive for floating labels -->
+                <!-- <span md-inputtext label="{{templateInfo.placeholder}}">
```

Comment:
I commented the md-inputtext directive implementation because it was not working and it was breaking the app.